### DISCUSSION TITLE
3.6.16355 pre-release (and 3.5) has a bug with the "Find" feature

### DISCUSSION THREAD

**Discussion by dc245y** [sumatrapdfreader/sumatrapdf#4331] [Q&A]
When searching for a word small rectangle appears with a progress bar.
But sometimes after a second or two it (the rectangle and the progress bar) just disappears while continuing a search. It then doesn't appear again until pressing "find next" or "find previous" button.
Solution - scroll one page down or up and launch the search again. Rectangle appears and doesn't disappear anymore

**Comment by GitHubRulesOK** [comment]
I am guessing you mean the find results box ? which is on a timeout
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/cb800b22-b27a-41e7-be82-30f746793ad0)

**Reply by dc245y** [reply]
Yes, exactly this
Can I disable this timeout so it doesn't disappear? It worked fine before on 3.4
